ons_alpha/articles/models.py

```diff
@@ -0,0 +1,187 @@
+from functools import cached_property
+
+from django.db import models
+from django.http import Http404
+from django.shortcuts import get_object_or_404, redirect
+from django.utils.translation import gettext as _
+from wagtail.admin.panels import (
+    FieldPanel,
+    FieldRowPanel,
+    HelpPanel,
+    MultiFieldPanel,
+    MultipleChooserPanel,
+    ObjectList,
+    TabbedInterface,
+)
+from wagtail.contrib.routable_page.models import RoutablePageMixin, path
+from wagtail.models import Page
+from wagtail.search import index
+
+from ons_alpha.bundles.models import BundledPageMixin
+from ons_alpha.core.blocks.stream_blocks import CoreStoryBlock, CorrectionsNoticesStoryBlock
+from ons_alpha.core.forms import PageWithUpdatesAdminForm
+from ons_alpha.core.models.base import BasePage
+from ons_alpha.utils.fields import StreamField
+
+
+class ArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):
+    base_form_class = PageWithUpdatesAdminForm
+    template = "templates/pages/article_page.html"
+    parent_page_types = ["ArticleSeriesPage"]
+    subpage_types = []
+
+    summary = models.TextField()
+    release_date = models.DateField()
+    next_release_date = models.DateField()
```

Comment:
Not a blocker. Articles don't seem to display a "next release date".

Reply:
it works on my machine?

![Screenshot 2024-07-26 at 09 30 56](https://github.com/user-attachments/assets/b677cc0c-0e18-40bc-9fd4-b5ccbad5329b)

Reply:
Sorry I wasn't clear. I mean I couldn't see this field on the live website. Have you got an example?

Reply:
https://www.ons.gov.uk/economy/governmentpublicsectorandtaxes/publicsectorfinance/articles/developmentofpublicsectorfinancestatistics/may2024